packages/Webkul/Admin/src/Resources/assets/js/components/navigation/notification.vue

```diff
@@ -145,6 +145,7 @@ export default {
         },
         readAll: function () {
             let this_this = this;
```

Comment:
Make this variable,

~~~js
let self = this;
~~~

If don't want to use `self` then use the `arrow` function.